app/main/main.js

```diff
@@ -67,6 +69,9 @@ function createWindow(tabInfo) {
   });
 
   browser.webContents.once('did-finish-load', () => {
+    let browserDidFinishLoadTime = new Date();
```

Comment:
I think `Date.now()` might be more intuitive, or could seem that we're passing in Date objects to the event method (when both Dates are cast to numbers anyway)